### ISSUE TITLE
Investigate using Landscape CNCF framework for FINOS Project Catalogue

### ISSUE THREAD

**Issue by mcleo-d** [finos/open-developer-platform#53]
## Description

finos.github.io was born as a simple way to deliver a visual overview of our projects. It was built with very basic knowledge and little time invested; the code is extremely basic, but it does the job. Nevertheless, it is quite heavy to improve, considering that the setup is extremely basic.

Looking around, there have been similar initiatives, the most similar to our requirements seems to be https://github.com/cncf/landscapeapp , which is used to render out https://landscape.cncf.io (even more relevant, for similarity in numbers, is https://landscape.lfai.foundation/ )

It would interesting to run a spike with https://github.com/cncf/landscapeapp :
- [ ] Consolidate FINOS taxonomy, see [Landscape Catalogue Analysis DRAFT](https://docs.google.com/document/d/1hLYPlYwandIMkjggNjCJ57a1UcY_iaVl0ncl8SnjXYg/edit#heading=h.okvxtya1vtaw) and [FINOS Landscape Taxonomy](https://docs.google.com/spreadsheets/d/18WqutqQmz1UvrAY_tR8caVUqnVus-gsBf4wc-PLblEM/edit#gid=0)
- [ ] Map `category` taxonomy in `landscape.yml` with data from [activities.json](https://github.com/finos/finos.github.io/blob/master/activities.json)
- [ ] Generate a `landscape.yml` for FINOS using metadata-tool
- [ ] Run the website locally and see the result
- [ ] (stretch goal) apply some FINOS styling

**Comment by brooklynrob** [comment]
I created[ this item in Jira](https://finosfoundation.atlassian.net/browse/INT-693) in June 2019 with assumption this work (putting finos.github.io on Docusaurus) would be an interim step that would improve and normalize the FINOS microsite/catalog (finos.github.io) with the standard way of building microsites used on FDC3, FO, etc. 

That said, the overall strategy/long term answer I think remains that the product catalog should just be an integrated part of the www.finos.org website, not a standalone web site. If that consolidation is something that's going to happen sooner than later, as I believe it might, this story would no longer make sense to considering doing or even to track. CC @toshaellison @grizzwolf

**Comment by maoo** [comment]
We discussed this today. We are looking into other solutions/technologies for finos.github.io , such as https://github.com/cncf/landscapeapp , the technology behind https://landscape.cncf.io . 

I will update this issue when we know more, until then let's keep it in TODO and low-prio.

**Comment by maoo** [comment]
Updated title and description!

**Comment by maoo** [comment]
@toshaellison - I managed to generate a `landscape.yml` from our project metadata, see https://gist.github.com/maoo/3d6f12a3403735458926a931a68f5268

I mapped programs as main category and the first tag as subcategory. Note that, up until now, we have never used tags, therefore:
- Changing them wont affect any other part of FINOS infra (we can play with them as we want
- Many of them are "null", so we'd need to set them depending on the taxonomy we eventually come up with
- We could also map the first category as a tag (Ie, the second tag in the list), if you'd prefer to map the main category to something else

Next step here is to see how it renders out, by running the landscape app with this dataset, and see what happens; at that point, we can consider this investigation task completed.

**Comment by mcleo-d** [comment]
Hi @maoo 

Thanks for moving forward with this ticket and providing the results of your investigation. I've moved this item from **Marketing** to **Prioritised** in the [FINOS Team Priorities](https://github.com/orgs/finos/projects/7#card-32218879) kanban.

Before moving fully into production, it would be great to get a team overview of https://landscape.cncf.io/ and what the work described in this ticket enables for FINOS.

Once the team intro is done, we can collectively move into **In Progress** 😄👍  

Speak soon,

James.

**Comment by maoo** [comment]
Given that we've done good progress in metadata-tool (we have a way to generate a landscape.yml file and push it into any FINOS GitHub repository) and that FINOS project taxonomy will be delivered by https://github.com/finos/open-developer-platform/issues/134 , I think we can close this issue.

As soon as the taxonomy is finalised, we will make changes to metadata and metadata-tool accordingly.
